### ISSUE TITLE
Support types from React@17 in react-property

### ISSUE THREAD

**Issue by AndrewLeedham** [remarkablemark/react-dom-core#14]
👋  
I am seeing the following error:
```
Warning: Invalid DOM property `disableremoteplayback`. Did you mean `disableRemotePlayback`?
```
when using `html-react-parser@1.2.7` to parse `<video disableremoteplayback></video>`, I traced it down to `react-property` which relies on `react-dom@15`, `disableRemotePlayback` looks to have been added in `react@17` here: https://github.com/facebook/react/commit/5f6b75dd265cd831d2c4e407c4580b9cd7d996f5

Having a look through the previous issues here it seems React internals changed so upgrading this package to `react@17` is not as simple as a dependency update. I am not sure what would be involved here, but just wanted to raise the issue with my findings. For now I can workaround the issue with a `replace` function.

**Comment by remarkablemark** [comment]
@AndrewLeedham thanks for opening this issue. Possible solutions are:

1. Clone the [react](https://github.com/facebook/react) repository since internals are no longer exposed
2. Keep a custom file of all DOM and SVG attributes

Any other ideas or suggestions?

**Comment by AndrewLeedham** [comment]
The other 2 options I can think of:
1. Raise an issue with react, to re-expose the internals as a supported API (not sure if this has already been attempted)
2. Do some-sort of scraping to get the source of the internals directly via GitHub, which throws an error if it can't find them so the path can be updated between versions if necessary. Perhaps it could be run as a cron job via GH actions?

**Comment by remarkablemark** [comment]
Thanks for coming up with the options @AndrewLeedham!

1. I think React is most likely not going backtrack from their decision of re-exposing internals. I remember seeing a lot of issues raised in the past and React said that they're not going to bring it back.
2. Scraping is an option but could get flaky if they move files around. Cloning `react` as a submodule might be more reliable.

**Comment by AndrewLeedham** [comment]
Ah cloning as a submodule, yeah that is a good idea!

**Comment by remarkablemark** [comment]
Released [react-property@2.0.0](https://github.com/remarkablemark/react-dom-core/releases/tag/react-property%402.0.0)

**Comment by remarkablemark** [comment]
https://github.com/remarkablemark/html-react-parser/pull/278
